# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/cmake/no-include-check.patch.cross0
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- cmake-3.20.5/Source/cmGeneratorTarget.cxx.vanilla	2021-07-22 15:50:56.490148603 +0200
+++ cmake-3.20.5/Source/cmGeneratorTarget.cxx	2021-07-22 16:22:37.592035289 +0200
@@ -3405,7 +3405,7 @@
     for (std::string& entryInclude : entry.Values) {
       if (fromImported && !cmSystemTools::FileExists(entryInclude)) {
         std::ostringstream e;
-        MessageType messageType = MessageType::FATAL_ERROR;
+        MessageType messageType = MessageType::AUTHOR_WARNING;
         if (checkCMP0027) {
           switch (tgt->GetPolicyStatusCMP0027()) {
             case cmPolicies::WARN:
@@ -3432,7 +3432,7 @@
              "does not provide.\n";
         /* clang-format on */
         tgt->GetLocalGenerator()->IssueMessage(messageType, e.str());
-        return;
+        //return;
       }
 
       if (!cmSystemTools::FileIsFullPath(entryInclude)) {
